Material.Blazor/Components/MenuSurface/MBMenuSurface.razor

```diff
@@ -0,0 +1,13 @@
+﻿@namespace Material.Blazor
+@inherits ComponentFoundation
+
+
+<div @ref="ElementReference"
+     @attributes="@AttributesToSplat()">
```

Comment:
To be replaced by 
```
class="mdc-menu-surface mdc-menu-surface--fixed @ActiveConditionalClasses @(@class)"
style="@style"
id="@id"
@attributes="@AttributesToSplat()"
```
when #421 is merged.

Reply:
Actually, `mdc-menu-surface--fixed` looks like it should be conditional.

Reply:
@HannahKiekens, please review whether `mdc-menu-surface--fixed` should be applied unconditionally or conditionally and just leave a comment here. I'll then adjust the code together with the changes necessary after #421 in one go.